### DISCUSSION TITLE
The direction of the world coordinate system is strange

### DISCUSSION THREAD

**Discussion by TomatoPicker278** [empriselab/RCareWorld#149] [Other Q&A] [answered]
Hi!
In the bathing scene, I attempted to translate the red box (ID 2333) in the x, y, and z directions. I encountered a puzzling issue: it seems that the world coordinate system in the bathing scene uses a left-handed coordinate system, rather than the more common right-handed coordinate system. Is this just my perception, or was it intentionally designed this way? Since I haven’t used Unity before, I'm not sure if this is a characteristic of Unity.

**Comment by YoruCathy** [comment, marked as answer]
Yes, in Unity, the default coordinate system is indeed a left-handed y-up coordinate system for 3D space. Feel free to use [this ](https://github.com/empriselab/RCareWorld/blob/phy-robo-care/pyrcareworld/pyrcareworld/utils/coordinate_system_converter.py)tool if you want to convert to other coordinate systems.

**Reply by TomatoPicker278** [reply]
Thank you very much!
